src/kemal/middleware/filters.cr

```diff
@@ -35,18 +47,25 @@ module Kemal::Middleware
     def initialize(&@block : HTTP::Server::Context -> _)
     end
   end
+
 end
 
 def add_filters
-  Kemal.config.add_handler Kemal::Middleware::Filter.new
+  unless filter = Kemal.config.handlers.any? { |handler| handler.is_a? Kemal::Middleware::Filter }
+    filter = Kemal::Middleware::Filter.new
```

Comment:
Ok cool so about the exception message, I see what you mean but I don't know how it should be done, I wanted the `path` to be interpolated in the exception message. How could this be moved inside the exception class?